content/en/docs/setup/install/multicluster/_index.md

```diff
@@ -10,12 +11,58 @@ keywords: [kubernetes,multicluster]
 test: n/a
 ---
 
-{{< tip >}}
-Note that these instructions are not mutually exclusive.
-In a large multicluster deployment, composed from more than two clusters,
-a combination of the approaches can be used. For example,
-two clusters might share a control plane while a third has its own.
-{{< /tip >}}
+This guide shows you how to deploy a multicluster
+{{< gloss >}}service mesh{{< /gloss >}}.
 
-Refer to the [multicluster deployment model](/docs/ops/deployment/deployment-models/#multiple-clusters)
-concept documentation for more information.
+Familiarity with [single cluster deployments](/docs/ops/deployment/deployment-models/#single-cluster)
+is needed before attempting a {{< gloss >}}multicluster{{< /gloss >}} deployment.
+
+To deploy a service mesh across multiple networks, go to the following legacy guides:
```

Comment:
Legacy means previous yet still valid, not deprecated. Technical writing best practice dictates that no time based information be included because it creates maintenance issues over time.